.github/workflows/main.yml

```diff
@@ -227,8 +294,8 @@ jobs:
         pip install virtualenv
         ./uhdm-integration/.github/ci.sh
         ./uhdm-integration/build/lowrisc_ibex_ibex_simple_system_0/sim-verilator/Vibex_simple_system --meminit=ram,uhdm-tests/ibex/hello_test.elf
-        mkdir output_uhdm
-        cp ibex_simple_system.log ibex_simple_system_pcount.csv trace_core_00000000.log output_uhdm/
+        mkdir output_uhdm_vanilla_verilator
+        cp ibex_simple_system.log ibex_simple_system_pcount.csv trace_core_00000000.log output_uhdm_vanilla_verilator/
```

Comment:
I suspect this change is by mistake? In ``simple-system-sim`` pipeline, we are not uploading vanilla verilator output, so ``output_uhdm`` folder is OK.

Reply:
Yep, right